dask/kubernetes/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/rapidsai/rapidsai-core:0.18-cuda11.0-runtime-ubuntu18.04-py3.8
+FROM docker.io/rapidsai/rapidsai-core:0.19-cuda11.0-runtime-ubuntu18.04-py3.8
```

Comment:
Update to latest version

Reply:
Can we just actually update to 21.06 then?